UndertaleModTool/Scripts/Resource Repackers/ReduceEmbeddedTexturePages.csx

```diff
@@ -273,9 +272,17 @@ public enum BestFitHeuristic
     MaxOneAxis,
 }
 
+public struct Rect

```

Comment:
this is like the fourth time that this is created in a script. It may be worth it to clarify that somewhere in the textureworker instead?